lib/improver/tests/utilities/test_cube_extraction.py

```diff
@@ -275,30 +350,30 @@ def setUp(self):
         """ Set up temporary input cube """
         self.precip_cube = set_up_precip_probability_cube()
 
-    def single_threshold(self):
+    def test_single_threshold(self):
```

Comment:
As mentioned, it would be useful to have a few more tests for the `extract_subcube` function to show the use of multiple constraints, so I would imagine a test with two constraints using the [value1,value2] syntax, a test with two constraints using the [value1:value2] syntax and a test with two constraints with one using the [value1,value2] syntax and the other using the [value1:value2] syntax.